scanner/api/op.cpp

```diff
@@ -90,7 +90,10 @@ OpRegistration::OpRegistration(const OpBuilder& builder) {
       new OpInfo(name, variadic_inputs, input_columns, output_columns,
                  can_stencil, stencil);
   OpRegistry* registry = get_op_registry();
-  registry->add_op(name, info);
+  Result result = registry->add_op(name, info);
+  if (!result.success()) {
```

Comment:
If you look at OpRegistry, I use the RESULT_ERROR macro which should print out a LOG(WARNING). I'm not sure how else to expose the failure since it's happening on library load.